tests/testColor.py

```diff
@@ -148,7 +148,7 @@ def testCalibFromCalibs(self):
         """Test creating a Calib from an array of Calibs"""
         mag0, mag0Sigma = 1.0, 0.01
 
-        calibs = afwImage.vectorCalib()
+        calibs = []
```

Comment:
This is a standard part of SWIG to pybind11 conversion. SWIG usually allowed either use of explicitly named vector templates or standard python lists, but pybind11 only supports the latter.